app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> str:
```

Comment:
The `get_rating` method should return an `int` instead of a `str`, as seen in the implementations of the subclasses. Ensure the return type matches across all implementations.